parm/config/gefs/config.ufs

```diff
@@ -117,19 +123,21 @@ case "${fv3_res}" in
         export fv_sg_adj=1800
         export WRITE_GROUP=1
         export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=1
-        export WRITE_GROUP_GFS=1
-        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=1
         ;;
     "C192")
+        if [[ "${RUN}" = *gfs ]]; then
+          export layout_x=4
+          export layout_y=6
+          export nthreads_fv3=2
+          export nthreads_ufs=2
+          export WRITE_GROUP=2
+          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=5
+        fi
         export DELTIM=600
         export layout_x=4
         export layout_y=6
-        export layout_x_gfs=4
-        export layout_y_gfs=6
         export nthreads_fv3=1
-        export nthreads_fv3_gfs=2
         export nthreads_ufs=1
-        export nthreads_ufs_gfs=2
```

Comment:
Here, the `nthreads_ufs` set in the `gfs` if block is being overridden by the following lines.  Suggest either having an if-else block here or declaring defaults first, then overriding with a RUN-based if block:

```suggestion
        export DELTIM=600
        export layout_x=4
        export layout_y=6
        export WRITE_GROUP=1
        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=10

        if [[ "${RUN}" = *gfs ]]; then
          export nthreads_fv3=2
          export nthreads_ufs=2
          export WRITE_GROUP=2
          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=5
        fi
```

Also, lines 151-4 should be deleted.